src/config/babelrc.js

```diff
@@ -55,7 +55,7 @@ module.exports = () => ({
       ['react', 'preact'],
       [
         require.resolve('@babel/preset-react'),
-        {pragma: isPreact ? 'React.h' : undefined},
```

Comment:
I'm pretty sure downshift relies on this. Could you verify this wouldn't break downshift? https://github.com/downshift-js/downshift

Reply:
I dug through the main code for downshift and didn't see any JSX used. I tested downshift with the updates and everything was fine. I couldn't run the `test:cypress` command using the standard `kcd-scripts` and the changed version, something about `A page component must export a React component for it to be valid`. Also I think (could be wrong) that it should work regardless if they're aliasing `react` to `preact/compat`.

Reply:
Oh I just realized, I don't think the pragma `React.h` would work anyway ... it doesn't exist?

React Top-Level API: https://reactjs.org/docs/react-api.html

Reply:
It works because of the alias

Reply:
Ah okay, that makes sense.

Reply:
But that's actually a good point. Maybe it's never actually used 😅

Reply:
The other option is I remove this specific change and for the preact testing library I also alias `react` to `preact/compat`. It just means all consumers of `kcd-scripts` will need to do the same and it won't work with the standard `Preact.h` JSX pragma. 

I guess ... to solve it for both cases we can read `package.json` and see if `preact/compat` is a dependency and use the correct pragma accordinly.

Reply:
Let's go with that second option. Thanks!